tests/components/life360/test_config_flow.py

```diff
@@ -118,7 +118,7 @@ async def test_user_show_form(hass, life360_api):
         assert keys[keys.index(key)].default == vol.UNDEFINED
 
 
-async def test_user_config_flow_success(hass, life360_api):
+async def test_user_config_flow_success(hass, life360_api, life360):
```

Comment:
The fixture is autoused.

https://github.com/home-assistant/core/blob/ce785e5b2511331a87e893b409f76cd0f6c7c7e9/tests/components/life360/test_config_flow.py#L67

Reply:
Yeah I noticed by a comment of @pnbruckner on the original PR, will correct/revert this.

Could not reproduce it, suspected what was in this PR, made a PR @ 2AM after a long day. Just missed that :(

But no, no stack trace. Only a message that test left dirty files behind in our CI (a stray know_devices.yaml file).

The life360 PR did match the time window.

Reply:
Opened up a PR to revert is, also removed it from the milestone.